test/org/zaproxy/zap/extension/pscanrulesBeta/CSRFCountermeasuresUnitTest.java

```diff
@@ -256,4 +258,71 @@ public void shouldRaiseLowAlertWhenFormAttributeAndRuleConfigMismatch() {
 		assertEquals(1, alertsRaised.size());
 		assertEquals(Alert.RISK_LOW, alertsRaised.get(0).getRisk());
 	}
+	
+	@Test
+	public void shouldNotRaiseAlertWhenThesholdHighAndMessageOutOfScope() throws URIException {
```

Comment:
There's a typo in the method names ("Threshold").

Reply:
Tks